packages/rs-drive-abci/tests/strategy_tests/main.rs

```diff
@@ -1159,6 +1159,7 @@ mod tests {
 
         let mut platform = TestPlatformBuilder::new()
             .with_config(config.clone())
+            .with_initial_protocol_version(PROTOCOL_VERSION_1)
```

Comment:
this test shouldn't need to set this.

Reply:
If you start with the latest version, the root hash will differ. And every time you will introduce a new version you will need to update hashes here